src/utils/database.js

```diff
@@ -88,11 +88,8 @@ function connect() {
 
 	const mongooseOptions = {
 		autoIndex: NODE_ENV !== ENVIRONMENTS.PRODUCTION,
-		poolSize: MONGOOSE_CONNECTION_POOL_SIZE,
+		minPoolSize: MONGOOSE_CONNECTION_POOL_SIZE, //https://mongoosejs.com/docs/migrating_to_6.html#mongodb-driver-40 ??
```

Comment:
> Mongoose 5.x poolSize option is equivalent to the Mongoose 6 maxPoolSize option. The default value of maxPoolSize has been increased to 100.

I guess we should add a maxPoolSize too. Default seems to be 100
In dof app we have   
```
MONGOOSE_CONNECTION_POOL_SIZE:
    value: "50"
```

so, we had a (max) 50 and now is min 50 to 100. Will that not cause any issues?